{{ cookiecutter.formal_name }}/app/src/main/java/org/beeware/android/MainActivity.java

```diff
@@ -139,17 +153,24 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
             Log.e("MainActivity", "Failed to create Python app", e);
```

Comment:
Should these historical logging references be updated to APP_LAUNCH_TAG?

Reply:
Seems smart, yeah!